### PR TITLE
Explicitly declare input and output files for svgo

### PR DESCRIPTION
**Pull request by Stack-of-Pancakes**
## TL;DR
SVG file minimization would fail when the input and output files were passed positionally.  Explicitly specifying input and output paths fixes this.

### Full explanation

Attempting to minify a SVG file was resulting in an error.

sublime error

When running the command from the CLI I was also getting an error

```powershell
Error: Error: no such file or directory 'C:\test\test.min.svg'.
    at checkOptimizeFileError (D:\nodejs\node_modules\svgo\lib\svgo\coa.js:465:31)
    at readFile.then.error (D:\nodejs\node_modules\svgo\lib\svgo\coa.js:378:18)
```
Current version of modules

module | version
--- | ---
cleancss | 4.1.11
uglifycss | uglifycss 0.0.29
js-beautify | 1.7.5
html-minifier | 3.5.16
uglifyjs | uglify-js 3.4.1
minjson | minjson v0.0.2
svgo | 1.0.5

Found that the problem was `svgo` wasn't accepting positional arguments for input and output files.  This was fixed with a positional input file and an argument specified output file.  Or just explicitly specifying input and output files.

Console results when specifying `-i` and `-o` parameters

```powershell
PS C:\test> svgo --version
1.0.5
PS C:\test> svgo
Nodejs-based tool for optimizing SVG vector graphics files

Usage:
  svgo [OPTIONS] [ARGS]


Options:
  -h, --help : Help
  -v, --version : Version
  -i INPUT, --input=INPUT : Input file, "-" for STDIN
  -s STRING, --string=STRING : Input SVG data string
  -f FOLDER, --folder=FOLDER : Input folder, optimize and rewrite all *.svg files
  -o OUTPUT, --output=OUTPUT : Output file or folder (by default the same as the input), "-" for STDOUT
  -p PRECISION, --precision=PRECISION : Set number of digits in the fractional part, overrides plugins p
arams
  --config=CONFIG : Config file or JSON string to extend or replace default
  --disable=DISABLE : Disable plugin by name
  --enable=ENABLE : Enable plugin by name
  --datauri=DATAURI : Output as Data URI string (base64, URI encoded or unencoded)
  --multipass : Enable multipass
  --pretty : Make SVG pretty printed
  --indent=INDENT : Indent number when pretty printing SVGs
  -q, --quiet : Only output error messages, not regular status messages
  --show-plugins : Show available plugins and exit

Arguments:
  INPUT : Alias to --input
PS C:\test> svgo C:\test\test.svg C:\test\test.min.svg
Error: Error: no such file or directory 'C:\test\test.min.svg'.
    at checkOptimizeFileError (D:\nodejs\node_modules\svgo\lib\svgo\coa.js:465:31)
    at readFile.then.error (D:\nodejs\node_modules\svgo\lib\svgo\coa.js:378:18)
PS C:\test> svgo C:\test\test.svg -o C:\test\test.min.svg

test.svg:
Done in 13 ms!
1.204 KiB - 58.3% = 0.502 KiB
PS C:\test> ls


    Directory: C:\test


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----        6/26/2018   1:43 PM            514 test.min.svg
-a----        6/26/2018  12:52 PM           1233 test.svg


PS C:\test> svgo -i C:\test\test.svg -o C:\test\test.min2.svg

test.svg:
Done in 13 ms!
1.204 KiB - 58.3% = 0.502 KiB
PS C:\test> ls


    Directory: C:\test


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----        6/26/2018   1:43 PM            514 test.min.svg
-a----        6/26/2018   1:43 PM            514 test.min2.svg
-a----        6/26/2018  12:52 PM           1233 test.svg
```

svgo error

### CODE REVIEW

**Comment by tssajo** [comment]
Thanks! I merged it
